clinica/pipelines/deeplearning_prepare_data/deeplearning_prepare_data_cli.py

```diff
@@ -0,0 +1,119 @@
+# coding: utf8
+
+
+import clinica.engine as ce
+
+
+class DeepLearningPrepareDataCLI(ce.CmdParser):
+
+    def define_name(self):
+        """Define the sub-command name to run this pipeline."""
+        self._name = 'deeplearning-prepare-data'
+
+    def define_description(self):
+        """Define a description of this pipeline."""
+        self._description = ('Tensor extraction (Pytorch) from T1w images:\n'
+                             'http://clinica.run/doc/Pipelines/DeepLearning_PrepareData/')
+
+    def define_options(self):
+        """Define the sub-command arguments."""
+        from clinica.engine.cmdparser import PIPELINE_CATEGORIES
+
+        # Clinica compulsory arguments (e.g. BIDS, CAPS, group_id...)
+        # Most of the time, you will want to read your pipeline inputs into
+        # a BIDS and/or CAPS directory. If your pipeline does not require BIDS input,
+        # simply remove the two lines involving the BIDS directory.
+        clinica_comp = self._args.add_argument_group(PIPELINE_CATEGORIES['CLINICA_COMPULSORY'])
+        clinica_comp.add_argument("caps_directory",
+                                  help='Path to the CAPS directory.')
+        clinica_comp.add_argument("tsv_file",
+                                  help='Path to the tsv file.')
+        clinica_comp.add_argument("extract_method",
+                                  help='''Method used to extract features. Three options:
+                                       'slice' to get 2D slices from the MRI,
+                                       'patch' to get 3D volumetric patches or
+                                       'whole' to get the complete MRI.''',
+                                  choices=['slice', 'patch', 'whole'], default='whole'
+                                  )
+
+        # group_id can be used by certain pipelines when some operations are performed at the group level
+        # (for example, generation of a template in pipeline t1-volume)
+        # clinica_comp.add_argument("group_id",
+        #                           help='User-defined identifier for the provided group of subjects.')
+
+        # Add your own pipeline command line arguments here to be used in the
+        # method below. Example below:
+        optional = self._args.add_argument_group(PIPELINE_CATEGORIES['OPTIONAL'])
+
+        optional.add_argument("use_full_MRI",
+                              help='''If this option is enable, the full MRI
+                                       is converted to tensor version. Otherwise
+                                       only the cropped version of the MRI is
+                                       converted.''',
+                              default=False, action="store_true"
+                              )
```

Comment:
I don't see its use in `run_command` method 🤔

Reply:
Fixed!